storage/mysql/map_storage.go

```diff
@@ -33,6 +33,9 @@ const (
 	selectLatestSignedMapRootSQL = `SELECT MapHeadTimestamp, RootHash, MapRevision, RootSignature, MapperData
 		 FROM MapHead WHERE TreeId=?
 		 ORDER BY MapHeadTimestamp DESC LIMIT 1`
+	selectGetSignedMapRootSQL = `SELECT MapHeadTimestamp, RootHash, MapRevision, RootSignature, MapperData
+		 FROM MapHead WHERE TreeId=? AND MapRevision=?
+		 ORDER BY MapHeadTimestamp DESC LIMIT 1`
```

Comment:
Don't think you need the order by clause as the above query should already restrict to one row. There a unique constraint on (TreeId, MapRevision).